### PR TITLE
[5.2] Add whereIn() and whereInLoose() to Collection

### PR DESCRIPTION
**Pull request by KennedyTedesco**
This is part of a use case i've had. Looks useful to me. :)

### CODE REVIEW

**Comment by themsaid** [comment]
I've been using Collections intensively for the past 3 days and I was about to suggest similar methods 👍

**Comment by garygreen** [comment]
+1

File: tests/Support/SupportCollectionTest.php [hunk 1]
```diff
@@ -273,6 +273,14 @@ public function testWhere()
         $this->assertEquals([['v' => 3], ['v' => '3']], $c->whereLoose('v', 3)->values()->all());
     }
 
+    public function testWhereIn()
+    {
+        $c = new Collection([['v' => 1], ['v' => 2], ['v' => 3], ['v' => '3'], ['v' => 4]]);
+
+        $this->assertEquals([['v' => 1], ['v' => 3]], $c->whereIn('v', [1, 3])->values()->all());
+        $this->assertEquals([['v' => 1], ['v' => 3], ['v' => '3']], $c->whereInLoose('v', [1, 3])->values()->all());
```
Comment (lucasmichot) [inline tests/Support/SupportCollectionTest.php]: I would put this test to a dedicated `testWhereInLoose`function

Comment (KennedyTedesco) [inline tests/Support/SupportCollectionTest.php]: It's a good idea. I'll do it ASAP.

**Pull request merged**